src/WorkItems/WorkItems.csproj

```diff
@@ -29,11 +30,11 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.2" />
     <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.20.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.2" />
-    <PackageReference Include="Microsoft.Json.Schema" Version="1.1.5" />
-    <PackageReference Include="Microsoft.Json.Schema.Validation" Version="1.1.5" />
+    <PackageReference Include="Microsoft.Json.Schema" Version="1.1.4" />
+    <PackageReference Include="Microsoft.Json.Schema.Validation" Version="1.1.4" />
     <PackageReference Include="Microsoft.TeamFoundationServer.Client" Version="16.170.0" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
><PackageReference Include="Newtonsoft.Json" Version="12.0.3" /> [](http://example.com/codeflow?start=4&length=63)

WorkItems doesn't need to be on v9.0?

Reply:
It can't run on it, due to our utilization of Microsoft.TeamFoundationServer.Client, which requires 12.x